net/curl/Portfile

```diff
@@ -352,10 +352,110 @@ subport curl-ca-bundle {
         set ca_bundle_dir ${prefix}/share/curl
         set openssl_dir ${prefix}/etc/openssl
         xinstall -d ${destroot}${ca_bundle_dir} ${destroot}${openssl_dir}
-        xinstall -m 644 ${worksrcpath}/lib/ca-bundle.crt ${destroot}${ca_bundle_dir}/curl-ca-bundle.crt
+        foreach dst [list mozilla-ca-bundle.crt curl-ca-bundle.crt] {
```

Comment:
`mozilla-ca-bundle.crt` is the downloaded CAs. `curl-ca-bundle.crt` serves as the merge file, I duplicate it here in case `activate` stage is interrupted by the user.